specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-02-preview/managedClusters.json

```diff
@@ -5781,6 +5788,17 @@
         }
       }
     },
+    "AgentPoolWindowsProfile": {
+      "type": "object",
+      "description": "The Windows agent pool's specific profile.",
+      "properties": {
+        "disableOutboundNat": {
+          "type": "boolean",
+          "title": "Whether to disable OutboundNAT in windows nodes",
+          "description": "The default value is false. Allow disabling OutboundNAT only if the cluster outbound type is NAT Gateway and the Windows agent pool doesn't have node public IP enabled."
```

Comment:
minor: Phrasing.

Change this to: ""Outbound NAT can only be disabled if the cluster outboundType is NAT Gateway and the Windows agent pool does not have node public IP enabled."